### ISSUE TITLE
Fix empty boxes on timeline

### ISSUE THREAD

**Issue by davidamichelson** [srophe/caesarea#325]
For some reason at https://caesarea-maritima.org/testimonia/browse.html?view=timeline There are two empty boxes without labels in them. They appear at the end after "Late Islamic Period". There are later periods in our data so perhaps the text is not displaying?

This is not a priority issue (so do the others first) and may require @wlpotter to help us know what is missing.

<img width="1051" alt="Screen Shot 2022-09-14 at 12 01 39 PM" src="https://user-images.githubusercontent.com/3083773/190217737-a7f94539-b3dd-441f-a8c0-92f14ddb9b6d.png">

**Comment by wlpotter** [comment]
I think those facets are pulling from this XML index file: https://github.com/srophe/caesarea/blob/master/documentation/caesarea-maritima-historical-era-taxonomy.xml ?

**Comment by wlpotter** [comment]
Oh weird. I will make a data issue for that.

Also https://github.com/srophe/caesarea-data/issues/99 might be to blame for the ones that are blank. We have a few cases where testimonia ended up with "NaN" in the origDate's attributes fields, resulting in blank `@period` attributes. Hopefully these two data errors are the culprit

**Comment by wlpotter** [comment]
FYI I fixed the issue with the hashtags in the period attributes, and one of the boxes went away (see attached screenshot).
I'm assuming the other box is from the issue with "NaN" dates, which we'll have to wait on until we get the correct dates back from the editor(s).

![Screenshot from 2022-09-16 11-19-48](https://user-images.githubusercontent.com/44096602/190684853-18a254ec-8766-489e-91ae-de6e732debc2.png)

**Comment by wlpotter** [comment]
@davidamichelson @wsalesky good news, https://github.com/srophe/caesarea-data/issues/99 was the culprit of the remaining empty box. This looks to be fixed now!
